sunpy/data/test/__init__.py

```diff
@@ -40,5 +41,26 @@ def get_test_filepath(filename, **kwargs):
     """
     return get_pkg_data_filename(filename, package="sunpy.data.test", **kwargs)
 
+def get_available_test_data():
+    """
+    Prints all available test data
+
+    """
+    file_name=""
+    for fn in get_pkg_data_filenames('test', 'sunpy.data','*'):
+        if(os.path.isdir(fn)):
+            for dir_contents in os.listdir(fn):
+                print (dir_contents)
+        else:
+            for file_path in reversed(fn):
+                if(file_path == '/'):
```

Comment:
@dpshelio i think i'll use os.name to check for the different os' and then use two if-blocks for the two different cases.